visualization/app/codeCharta/ui/rangeSlider/rangeSlider.component.ts

```diff
@@ -71,10 +72,10 @@ export class RangeSliderController implements SettingsServiceSubscriber {
 	}
 
 	private updateInputFieldWidth() {
-		let fromLength = this.settingsService.settings.neutralColorRange.from.toFixed().toString().length
-		let toLength = this.settingsService.settings.neutralColorRange.to.toFixed().toString().length
-		let fromWidth = Math.max((fromLength + 1) * this.DIGIT_LENGTH, this.STANDARD_WIDTH_RANGE_INPUT)
-		let toWidth = Math.max((toLength + 1) * this.DIGIT_LENGTH, this.STANDARD_WIDTH_RANGE_INPUT)
+		let fromLength = this.settingsService.settings.neutralColorRange.from.toFixed().toString().length + 1
```

Comment:
Do we actually need the `toFixed()`?

Reply:
I added this there because when a map is loaded, sometimes the calculated values are not whole numbers/integers. This can cause the inputFields to be to wide when opening the ribbon bar for the first time after import.

Reply:
Alright ;)